src/data/fintype/basic.lean

```diff
@@ -229,6 +229,76 @@ def of_surjective [decidable_eq β] [fintype α] (f : α → β) (H : function.s
   fintype β :=
 ⟨univ.image f, λ b, let ⟨a, e⟩ := H b in e ▸ mem_image_of_mem _ (mem_univ _)⟩
 
+end fintype
+
+section inv
+
+variables [fintype α]
+
+lemma function.injective.exists_unique_fintype_of_mem_range {f : α → β} (hf : function.injective f)
+  {b : β} (hb : b ∈ set.range f) : ∃! a, a ∈ (finset.univ : finset α) ∧ f a = b :=
+begin
+  obtain ⟨a, rfl, ha⟩ := hf.exists_unique_of_mem_range hb,
+  exact ⟨a, ⟨finset.mem_univ a, rfl⟩, λ a' ha', ha a' ha'.right⟩
+end
```

Comment:
This lemma feels a bit silly to me, since it just inserts an always-true statement in the middle of an interesting lemma.

I'd suggest just inlining it below...